ci/pipelines/toolbox-staging-ecr.yml

```diff
@@ -0,0 +1,165 @@
+resources:
+  - name: toolbox-staging-ecr
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master 
+      paths:
+        - ci/pipelines/toolbox-test-ecr.yml
+  # ECR Test registry resource
+  - name: ecr-registry-test
+    type: dev-registry-image
+    icon: docker
+    source:
+      repository: govukpay/toolbox
+      aws_access_key_id: ((readonly_access_key_id))
+      aws_secret_access_key: ((readonly_secret_access_key))
+      aws_session_token: ((readonly_session_token))
+      # Need to create the role for the concourse user to assume in the test account
+      aws_role_arn: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+      # Hardcode the test account registry ID for now. Needs to be a string, not a number
```

Comment:
I guess this comment can go, looks like its no longer hard coded and is taken from concourse's parameters/ssm?